Makefile.Downstream.mk

```diff
@@ -1,6 +1,6 @@
 include Makefile
 
-BUNDLE_IMAGE_NAME ?= $(IMAGE_NAME)-bundle
+BUNDLE_IMAGE_NAME ?= cephcsi-operator-bundle
 
```

Comment:
For downstream image we set it as cephcsi-operator and push that image to ocs-dev quay.io, IMO we don't need this change

Reply:
currently, action is building the image as `ceph-csi-operator-bundle` which is problematic for odf upstream as odf needs the same name as package name. Thats why we need a change or else I have to export the name in the action itself.

Reply:
Yes makes sense to export the image name in the action(the action is already downstream specific) to get the image with the same downstream name and also the bundle image name.

Reply:
Done